src/bin/radclient.c

```diff
@@ -1228,6 +1229,14 @@ int main(int argc, char **argv)
 			if (resend_count < 1) usage();
 			break;
 
+		case 'C':
+			if (!isdigit((int) *optarg)) usage();
+
+			client_port = atoi(optarg);
```

Comment:
this should be assigned to an intermediate variable, because `client_port` is 16 bits, and there can be integer overflow.

Reply:
Done.